examples/pytorch/AxHyperOptimizationPTL/conda.yaml

```diff
@@ -0,0 +1,19 @@
+channels:
+- defaults
+- conda-forge
+- pytorch
+dependencies:
+- python=3.8.2
+- pytorch>=1.6.0
+- torchvision
+- cudatoolkit>=10.2
```

Comment:
Do we explicitly need to install `cudatoolkit`?

Reply:
i have removed it.